recipes/mfront/meta.yaml

```diff
@@ -0,0 +1,79 @@
+{% set name = "mfront" %}
+{% set version = "3.2.1" %}
+{% set sha256 = "12786480524a7fe86889120fb334fa00211dfd44ad5ec71e2279e7adf1ddc807" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}_{{ version }}.tar.gz
+  url: https://github.com/thelfer/tfel/archive/refs/tags/TFEL-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  skip: true  # [py2k or osx or win or (python_impl == 'pypy')]
+  detect_binary_files_with_prefix: True
+  run_exports:
+    - {{ pin_subpackage('mfront', max_pin='x.x.x') }}
+    - {{ pin_compatible('python', max_pin='x.x') }}
+
+requirements:
+  build:
+    - gcc 8  # [linux]
+    - gxx 8  # [linux]
+    - gfortran 8  # [linux]
```

Comment:
Indeed I could not compile this aster version using more recent compilers, so I kept these ones for mfront since aster will link at mfront. I am confident that newer versions will work on more recent compilers and I will remove this constraint (some projects I plan to submit need this specific version)